### DISCUSSION TITLE
Option to add more filterlist for incognito mode

### DISCUSSION THREAD

**Discussion by LennyFox** [uBlockOrigin/uBlock-issues#2590] [Ideas]
When a user starts an incognito windows, uBO by design loads a copy of uBO specifically for the incognito mode.

Edge has an option to use STRICT mode for anti-tracking when the user opens an inprivate (=incognito) window. From the user perspective this makes sense. When you want to surf incognito, you would like your anti-tracking protection level to rise (increase to a higher level of protection).

Would it be possible to have two sets of filterlist selections? One for normal mode and one for incognito mode. The use case for adding more filter lists when switching to incognito mode seems logical and valid. Several browsers have this option already, so it seems to make sense to me that extensions copy this behavior.

**Comment by gothic-bum** [comment]


**Reply by Kees1958** [reply]
And what purpose does not saving data between sessions has?  Why were tracking-cookies stored in the first place. How  are all applications labeled/named which remove the recently used entries in Windows? Come on...

**Reply by LennyFox** [reply]
I now have manualy added allow exceptions for the websites I wanted to sponsor with advertisements, so you can close my suggestion. I will add this comment also in my suggestion,

**Reply by Kees1958** [reply]
From the Mozilla/Firefox link you provides  :-)

**Or maybe you just want to limit the amount of data companies collect about you and you value privacy. I**ncognito or private browsing mode is made for any of these scenarios.

**Reply by iam-py-test** [reply]
Be aware that for the most part, "private browsing" just doesn't save cookies/local storage/history (and in Firefox and Edge, uses stricter tracking protection). It really don't provide much more privacy/security

**Reply by Kees1958** [reply]
Yes, I know that but Edge has two options (one for security and one for privacy) which users can enable when going incognito (=inprivate). Firefox has an option in anti-tracking to block trackers in all tabs or only inprivate tabs. Chrome has the option to set additional website permission (when incognito mode) for all sessions or only for this incognito session (temporary rule). 

In Edge those options are shown on the inprivate new tab. When three browsers have developed features which allow for more STRICT (see first picture), privacy or security or both (site permissions), while the initial use (chromium) was to hide stuff from other users on the same PC and Firefox explicitly mentions this second use (see previous post), there are more people found a need to add this use case. 

![image](https://user-images.githubusercontent.com/30925856/235589678-40c8b8ba-6e5e-4344-98fb-3d27451b2291.png)

**Reply by iam-py-test** [reply]
I understand, and it is an interesting idea, I'm just not sure how easy it is to implement in uBo without changing how it works. But, that's up to gorhill, I have no say in it.
